addons/account/tests/test_chart_template.py

```diff
@@ -267,3 +268,53 @@ def test_message_to_accountants(self):
             ('partner_ids', 'in', normal_user.partner_id.ids),
             ('body', 'like', f"%{self.tax_template_1.name}%"),  # we look for taxes' name that have been sent in the message's body
         ]), 0)
+
+    def test_update_taxes_foreign_taxes_(self):
+        """ When taxes are instantiated through the fiscal position system (in multivat),
+        its taxes should also be updated.
+        """
+        country_test = self.env['res.country'].create({
+            'name': 'Country Test',
+            'code': 'ZZ',
+        })
+        chart_template_xmlid_test = 'l10n_test.test_chart_template_xmlid'
+        chart_template_test = self.env['account.chart.template']._load_records([{
+            'xml_id': chart_template_xmlid_test,
+            'values': {
+                'name': 'Test Chart Template',
+                'currency_id': self.env.ref('base.EUR').id,
+                'bank_account_code_prefix': 1000,
+                'cash_account_code_prefix': 2000,
+                'transfer_account_code_prefix': 3000,
+                'country_id': country_test.id,
+            }
+        }])
+        self._create_tax_template('account.test_tax_test_template', 'Tax name 1 TEST', 10, chart_template_id=chart_template_test.id)
+        self.env['account.tax.template']._try_instantiating_foreign_taxes(country_test, self.company)
+        tax_template_test_2 = self._create_tax_template('account.test_tax_test_template2', 'Tax name 2 TEST', 15, chart_template_id=chart_template_test.id)
+        update_taxes_from_templates(self.env.cr, chart_template_xmlid_test)
+
+        tax_test = self.env['account.tax'].search([
+            ('company_id', '=', self.company.id),
+            ('name', '=', tax_template_test_2.name),
+        ])
```

Comment:
Rather search for it with the xmlid generated at template instantiation. It's always better not to depend on the names ^^